### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by kristofmuhi**
new iOS17 update related to supported features.
This will go away in 2-3 weeks

### CODE REVIEW

File: docs/mobile-apps/faq.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
 - **Android Real Devices** - Yes. It works automatically with no action required.
 - **iOS Real Devices** - Yes. Prior to testing, you'll need to disable resigning; this enables notifications testing. Disabling resigning is a feature available for [private devices only](/mobile-apps/supported-devices#private-device-cloud). To disable resigning, go to **Live** > **Mobile App** > Locate your iOS mobile app from your list of uploaded app > Click **Settings** > Toggle **Instrumentation** so that it says **Disabled**. You'll need to start keeping track of the iOS device UDIDs (Unique Device Identifier) by maintaining them in your own Apple Developer profile used at app build time.
 - **Android Emulators** - not supported.
```
Comment (wswebcreation) [inline docs/mobile-apps/faq.md:53]:
```suggestion
- **Android Emulators** - Not supported.
```

File: docs/mobile-apps/faq.md [hunk 2]
```diff
@@ -50,8 +50,8 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
...
 - **iOS Real Devices** - Yes. Prior to testing, you'll need to disable resigning; this enables notifications testing. Disabling resigning is a feature available for [private devices only](/mobile-apps/supported-devices#private-device-cloud). To disable resigning, go to **Live** > **Mobile App** > Locate your iOS mobile app from your list of uploaded app > Click **Settings** > Toggle **Instrumentation** so that it says **Disabled**. You'll need to start keeping track of the iOS device UDIDs (Unique Device Identifier) by maintaining them in your own Apple Developer profile used at app build time.
-- **Android Emulators** - not supported.
-- **iOS simulators** - Yes, when using Appium version 1.21.0. See [Announcing support for Appium v1.21.0 on iOS Simulators](https://changelog.saucelabs.com/en/announcing-support-for-appium-v-on-ios-simulators) for more information.
+- **Android Emulators** - Not supported.
+- **iOS simulators** - Yes, when using the latest Appium version.  See [Announcing support for Appium v1.21.0 on iOS Simulators](https://changelog.saucelabs.com/en/announcing-support-for-appium-v-on-ios-simulators) for more information.
```
Comment (wswebcreation) [inline docs/mobile-apps/faq.md:54]:
```suggestion
- **iOS simulators** - Yes, when using the latest Appium version. 
```

File: docs/mobile-apps/faq.md [hunk 3]
```diff
@@ -96,7 +96,7 @@ Yes. We support OAuth login via Google and GitHub.
 
 #### **Are your devices jailbroken?**
 
-No. We do not offer jailbroken or rooted devices.
+No. We do not offer jailbroken or rooted devices, or penetration testing.
```
Comment (wswebcreation) [inline docs/mobile-apps/faq.md:99]: @kristofmuhi 

I'm assuming you mean we don't support "penetration testing", if so, can you change this sentence a bit/remove this and place it somewhere else? It's a bit out of context now

File: docs/mobile-apps/faq.md [hunk 4]
```diff
@@ -96,7 +96,7 @@ Yes. We support OAuth login via Google and GitHub.
...
```
Comment (wswebcreation) [inline docs/mobile-apps/faq.md:99]: @kristofmuhi 

What do you exactly mean with this, I assume it's not supported. If that's the case then can you change the sentence/place it in a separate question so it matches the context?

**Review by wswebcreation — COMMENTED** [review]
@kristofmuhi 

Added 1 comment and fixed some stuff, can you take a look at it again?

File: docs/mobile-apps/faq.md (hunk 3 above)
Comment (kristofmuhi) [reply docs/mobile-apps/faq.md]: reverted

**Comment by kristofmuhi** [comment]
reviewed with Wim

**Review by wswebcreation — APPROVED** [review]
LGTM

**Pull request merged**